.changeset/plenty-eyes-develop.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': patch
```

Comment:
I know this is a bugfix, but if folks are relying on workarounds currently, this patch will break them. Should this be a `minor` to be safe?

Reply:
Yeah I'd be ok with that.